src/main/java/io/vertx/mqtt/MqttClientOptions.java

```diff
@@ -294,6 +297,36 @@ public MqttClientOptions setKeepAliveTimeSeconds(int keepAliveTimeSeconds) {
     return this;
   }
 
+  /**
+   * Sets the time after which the client stops waiting for a PUBACK or PUBREC
```

Comment:
_Sets the time in seconds_

Reply:
The unit is already defined in the parameter description by I can, of course, put it here as well ...